server/Cargo.toml

```diff
@@ -28,7 +28,7 @@ itertools = "0.10.1"
 juniper = "0.15.10"
 juniper_actix = "0.4.0"
 jwt = "0.13"
-ldap3_proto = "*"
+ldap3_proto = { git = "https://github.com/kanidm/ldap3", package = "ldap3_proto" }
```

Comment:
I'd rather pin it to a specific commit hash than depend on master.

Also, you can do that with an override in the top-level Cargo.toml.